public/playersBetController.js

```diff
@@ -0,0 +1,157 @@
+      this.controlsContainerEle.querySelector(".bet-controls__pot");
+    this.curretBetCont =
+      this.controlsContainerEle.querySelector(".bet-controls__bet");
+    this.confirm = this.controlsContainerEle.querySelector(".btn--bet");
+  }
+
+  addEventListeners() {
+    this.bet25.addEventListener("click", this.betting.bind(this));
+    this.bet50.addEventListener("click", this.betting.bind(this));
+    this.bet100.addEventListener("click", this.betting.bind(this));
+    this.clear.addEventListener("click", this.clearBet.bind(this));
+    this.confirm.addEventListener("click", this.confirmBet.bind(this));
+  }
+
+  updateActivePlayer() {
+    this.activePlayer = this.players[this.currentPlayerTurn];
+    document
+      .querySelector(`#${this.activePlayer.name} .card`)
+      .classList.add("active");
+  }
+
+  isLastPlayerTurn() {
+    const lastPlayerPosition = this.players.length - 2;
+    return lastPlayerPosition === this.currentPlayerTurn;
+  }
+
+  nextPlayer() {
+    document
+      .querySelector(`#${this.activePlayer.name} .card`)
+      .classList.remove("active");
+    if (this.isLastPlayerTurn()) {
+      this.betControls.remove();
+      this.board.startWithGameDealing();
+    } else {
+      this.currentPlayerTurn++;
+      this.updateActivePlayer();
+      this.currentPotCont.innerHTML = `Pot: ${this.activePlayer.pot}`;
+      this.curretBetCont.innerHTML = `Current bet: ${this.currentBet}`;
+    }
+  }
+
+  betting(e) {
+    this.clear.disabled = false;
+    this.confirm.disabled = false;
+    const betAddition = e.target.innerText;
+
+    if (this.activePlayer.pot - betAddition < 0) {
+      return;
+    }
+
+    this.activePlayer.removeFromPot(betAddition);
+    this.currentBet += Number(betAddition);
+    this.currentPotCont.innerHTML = `Pot: ${this.activePlayer.pot}`;
+    this.curretBetCont.innerHTML = `Current bet: ${this.currentBet}`;
+
+    switch (true) {
+      case this.activePlayer.pot < 25:
+        this.bet25.setAttribute("disabled", "true");
+      case this.activePlayer.pot < 50:
+        this.bet50.setAttribute("disabled", "true");
+      case this.activePlayer.pot < 100:
+        this.bet100.setAttribute("disabled", "true");
+    }
+  }
+
+  clearBet() {
+    //is there a better way to do this ?
+    this.enableBettingButtons();
+
+    this.clear.disabled = true;
+    this.confirm.disabled = true;
+    this.activePlayer.pot += this.currentBet;
+    this.currentBet = 0;
+    this.currentPotCont.innerHTML = `Pot: ${this.activePlayer.pot}`;
+    this.curretBetCont.innerHTML = `Current bet: ${this.currentBet}`;
+  }
+
+  confirmBet() {
+    this.enableBettingButtons();
+    this.clear.disabled = true;
+    this.confirm.disabled = true;
+    this.activePlayer.renderBetValue(this.currentBet);
+    this.currentBet = 0;
+    this.nextPlayer();
+  }
+
+  enableBettingButtons() {
+    this.bet25.removeAttribute("disabled");
+    this.bet50.removeAttribute("disabled");
+    this.bet100.removeAttribute("disabled");
```

Comment:
As an additional challenge you could see if it is possible to dynamically generate or query the bet buttons